src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -44,7 +44,7 @@
     private final TransactionControl txControl;
     private final EntityManager em;
 
-    private List<DeviceMemberEntity> validateDeviceMembers(Long parentId, Long vsId, Long memberId, String childName) {
+    private List<DeviceMemberEntity> getDeviceMembers(Long parentId, Long vsId, Long memberId, String childName) {
```

Comment:
rename parentId to deviceId.

DeviceMemberEntity should also have the parent field renamed to deviceId. 

Using parent/child etc is fine when we are trying to be generic(at a base object) but it becomes confusing at the child objects to refer to them that way.

Reply:
yes will do that. I had followed this approach by looking at the some of the OSC and plugin entities which still contains parentId fields (for example PodPort).
Will clean this up.